rust-tide-backend/src/app.rs

```diff
@@ -0,0 +1,44 @@
+use crate::{error::Error::*, handler, settings::logging, Result, CONFIG};
+use log::info;
+
+pub struct AppState {
+    pub sample_state: String,
+}
+
+impl AppState {
+    pub fn new() -> Self {
+        AppState {
+            sample_state: String::from("dummy state"),
+        }
+    }
+}
+
+pub struct App {}
+
+impl App {
+    pub fn new() -> Result<App> {
```

Comment:
```suggestion
    pub fn new() -> App {
```
or is it on purpose for future changes?

Reply:
copied over as well - but it might well be that some error-able initialization comes in here at some point. But it could be removed for now for sure 👍